src/terminal/parser/InputStateMachineEngine.hpp

```diff
@@ -125,68 +118,43 @@ namespace Microsoft::Console::VirtualTerminal
             F12 = 24,
         };
 
-        struct CSI_TO_VKEY
-        {
-            CsiActionCodes Action;
-            short vkey;
-        };
+        static const std::map<CsiActionCodes, short> CsiMap;

```

Comment:
NAK on std::map. Use a constexpr `std::array` with aggregate initialization like in CodepointWidthDetector.

Reply:
OK, per our verbal discussion, will fix it.